.github/workflows/ci.yml

```diff
@@ -30,7 +30,3 @@ jobs:
       run: npm run test
     - name: Build
       run: npm run build
-    - name: Upload Coverage
-      uses: codecov/codecov-action@v3
-      with:
-        fail_ci_if_error: true
```

Comment:
Why was this removed?

Reply:
codecov was being flaky and comparing my commit against changes from 6 commits ago.  I'll put it back after I merge this.